bin/makeMarkerStats.R

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env Rscript
+
+suppressPackageStartupMessages(require(optparse))
+
+option_list = list(
+  make_option(
+    c("-c", "--counts-dir"),
+    action = "store",
+    default = NA,
+    type = 'character',
+    help = "Path to old-school 10X dir with barcodes.tsv, genes.tsv and matrix.mtx, for count data"
+  ),
+  make_option(
```

Comment:
we will need to produce I guess "cluster" files for cell type or any other grouping, correct? This is just a simple tabular file with assignments, correc?

Reply:
I'm taking the annotation-derived groupings directly from an SDRF-derived tabular file, any manner of grouping could be in there. Right now the clusters file is just for the unsupervised clusterings.